middle.mkd

```diff
@@ -401,25 +398,16 @@ possible for this array to be empty.
 
 # Coordinate Reference System
 
-The default reference system for all GeoJSON coordinates SHALL be
+The coordinate reference system for all GeoJSON coordinates SHALL be
```

Comment:
s/SHALL be/is/

This only needs to be a simple statement of fact.  The interoperability requirement is the one at the end of the paragraph.